openshift_scalability/content/quickstarts/stress/stress-pod.json

```diff
@@ -35,6 +35,10 @@
                                 "value": "${STRESS_CPU}"
                             },
                             {
+                                "name": "ROUTER_IP",
```

Comment:
Just moved this up a few lines?

Reply:
Yes, actually, it is quite a few lines up.  Moving closer towards JMeter tests, as mb tests don't need it.